client/devdocs/index.js

```diff
@@ -15,7 +15,7 @@ export default function() {
 		page( '/devdocs/form-state-examples/:component?', controller.sidebar, controller.formStateExamples );
 		page( '/devdocs/design/typography', controller.sidebar, controller.typography );
 		page( '/devdocs/design/:component?', controller.sidebar, controller.design );
-		page( '/devdocs/app-components/:component?', controller.sidebar, controller.blocks );
+		page( '/devdocs/blocks/:component?', controller.sidebar, controller.blocks );
```

Comment:
I don’t think we should have renamed this occurrence since it was ensuring the old URLs worked.